lib/editors/org_editor.dart

```diff
@@ -1,3 +1,18 @@
+/*
+Copyright 2020-2021 Vishesh Handa <me@vhanda.in>
```

Comment:
You probably want to change the name :D

Reply:
I did it for this file. Not even sure how this works. :@